frontend/src/app/pages/exercises/exercise/shared/editor-panel/create-image-template-modal/create-image-template-modal.component.ts

```diff
@@ -32,22 +32,21 @@ export class CreateImageTemplateModalComponent implements OnDestroy {
     ) {}
 
     public async createImageTemplate() {
```

Comment:
```suggestion
    public createImageTemplate() {
```

I don't think this is needed anymore.

Reply:
Thumbs up but no change?